docs/agents/sematext-agent/processes/metrics.md

```diff
@@ -0,0 +1,65 @@
+title: Process Monitoring
+description: Process Monitoring memory CPU usage
+
+Sematext Agent gathers various metrics and metadata about running processes and brings them to [Sematext Cloud](https://sematext.com/cloud/) so you can painlessly explore the system resources allocated by processes whether they're executing on bare metal machines or scheduled to run on multi-cluster Kubernetes workloads.
+
+Process collector is enabled by default. You can disable the collection of the process's metrics/metadata by setting the following properties:
```

Comment:
You can disable the collection of the process's metrics/metadata by => You can disable process metrics/metadata collection by